Marlin/src/HAL/LPC1768/HAL_SPI.cpp

```diff
@@ -358,7 +358,7 @@ void SPIClass::updateSettings() {
   //SSP_DeInit(_currentSetting->spi_d); //todo: need force de init?!
 
   // divide PCLK by 2 for SSP0
-  CLKPWR_SetPCLKDiv(_currentSetting->spi_d == LPC_SSP0 ? CLKPWR_PCLKSEL_SSP0 : CLKPWR_PCLKSEL_SSP1, CLKPWR_PCLKSEL_CCLK_DIV_2);
+  // CLKPWR_SetPCLKDiv(_currentSetting->spi_d == LPC_SSP0 ? CLKPWR_PCLKSEL_SSP0 : CLKPWR_PCLKSEL_SSP1, CLKPWR_PCLKSEL_CCLK_DIV_2);
```

Comment:
Does this end up impacting SD card access or non-TFT SPI displays?

How confident that nothing is breaking for those on common boards?

Reply:
I tested in a board that the LPC framework can't use the SD because it's to fast... So, I think it will work in the other boards that the LPC framework already works at 25Mhz!
Marlin SPI is limited (currently) at 8Mhz! In a 32bit boart that the SPI can go further than 30Mhz, that dividir might not e a problem.

Reply:
> I tested in a board that the LPC framework can't use the SD

So there are currently problems with this PR and LPC SD?

I might be misunderstanding your comment.

Reply:
the LPC framework read the SD using SPI speed at 25mhz, for MSC.... But, I just found a board that it don't work because that speed. It's related with this issue: https://github.com/MarlinFirmware/Marlin/pull/19939 ..
So, all other boards that use LPC framework in marlin already works at 25Mhz to read the SD.
In marlin space it's limited to 8mhz and worked in my board that have issue with the LPC framework.

The conclusion, for me, is that we are using SPI speed in LPC at very slow speed, than we really need.

Reply:
In resume: We are working 3x slower than we already can for SPI SD speed (because LPC framework already uses SPI speed 3x higher than marlin space), so removing that clock divisor won't do any harm.

Reply:
I will take a look if we have any hw spi tft in LPC. If we have, I don't think it will impact either.

Reply:
I found two LCD using hw SPI on LPC. One explicitly set the speed. The other just use the speed from SD. So, both might work. Maybe ST7920 could be affect. I'm not sure, as it set the speed at the lowest.

Reply:
Just tested on SKR 1.4 Turbo (LPC1769) and FYSETC_GENERIC_12864_1_1. It works without any issues (regardless FORCE_SOFT_SPI).